apps/testingaccessibility/src/pages/api/feedback.ts

```diff
@@ -0,0 +1,75 @@
+import type {NextApiRequest, NextApiResponse} from 'next'
+import {withSentry} from '@sentry/nextjs'
+import * as Sentry from '@sentry/nextjs'
+import {setupHttpTracing} from '@vercel/tracing-js'
+import {tracer} from '../../utils/honeycomb-tracer'
+import {getDecodedToken} from '../../utils/get-decoded-token'
+import nodemailer from 'nodemailer'
+import postmarkTransport from 'nodemailer-postmark-transport'
+import prisma from '../../db'
+import parseMarkdown from 'markdown-it'
+import removeMarkdown from 'remove-markdown'
+
+const transport = nodemailer.createTransport(
+  postmarkTransport({
+    auth: {
+      apiKey: process.env.POSTMARK_KEY,
+    },
+  }),
+)
+
+const feedbackHandler = async (req: NextApiRequest, res: NextApiResponse) => {
+  setupHttpTracing({name: feedbackHandler.name, tracer, req, res})
+  if (req.method === 'POST') {
+    try {
+      const token = await getDecodedToken(req)
+      const user = await prisma.user.findUnique({
+        where: {
+          id: token?.sub,
+        },
+      })
+
+      if (!user) {
+        res.status(403).end('not authorized')
+      } else {
+        const comment = await prisma.comment.create({
+          data: {
+            userId: user.id,
+            text: req.body.text,
+            context: req.body.context,
+            updatedAt: new Date(),
+          },
+        })
+
+        const info = await transport.sendMail({
+          from: `Testing Accessibility Feedback <${process.env.NEXT_PUBLIC_SUPPORT_EMAIL}>`,
+          to: process.env.NEXT_PUBLIC_SUPPORT_EMAIL,
+          replyTo: user.email,
+          subject: `Feedback from ${
+            user.name ? user.name : user.email
+          } about Testing Accessibility`,
+          text: removeMarkdown(comment.text),
+          html: parseMarkdown().render(comment.text),
+        })
```

Comment:
send the email via Postmark

uses markdown, but I didn't configure stuff like emoji, syntax highlights, etc

`markdown-it` has a ton of options -> https://github.com/markdown-it/markdown-it